tests/ops/service/connectors/fides/test_fides_client.py

```diff
@@ -88,7 +87,7 @@ def test_authenticated_request_not_logged_in(self, test_fides_client: FidesClien
         assert "No token" in str(exc)
 
     @mock.patch(
-        "requests.post",
+        "httpx.post",
```

Comment:
If the tests are passing I think we're ok there?

The reasoning here is as follows:
1. General tech debt/maintenance. FastAPI is a critical part of the application and we want to keep relatively on top of new releases
2. There is a critical dependency change we want to make in `fidesplus` that _requires_ a newer version of `FastAPI`, so this is considered customer-required work
3. `httpx` is a newer library specifically designed around Python's async/await internals and so is a better choice from a technical standpoint.

We still use `requests` in other parts of the application, so we aren't wholesale replacing `requests` everywhere. In this case though the amount of work to refactor the tests vs. directly replacing `requests` it felt like replacing `requests` was the better move in the long and short term.

Reply:
just noting that @ThomasLaPiana and i synced offline and we're going to try and trigger external saas requests since that's where a lot of the test coverage is for `saas.AuthenticatedClient`, just to be sure we haven't accidentally missed something here.